src/gt4sd/training_pipelines/tests/test_training_guacamol_baselines_smiles_lstm_hc.py

```diff
@@ -0,0 +1,46 @@
+"""Language modeling trainer unit tests."""
+
+from typing import Any, Dict, cast
+
+import pkg_resources
+
+from gt4sd.training_pipelines import (
+    TRAINING_PIPELINE_MAPPING,
+    GuacamolLSTMHCTrainingPipeline,
+)
+
+template_config = {
+    "model_args": {
+        "batch_size": 512,
+        "valid_every": 1000,
+        "n_epochs": 10,
+        "hidden_size": 512,
+        "n_layers": 2,
+        "rnn_dropout": 0.2,
+        "lr": 1e-3,
+    },
+    "training_args": {
+        "max_len": 100,
+        "output_dir": "/Users/ashishdave/Desktop/GT4SD/gt4sd-core/",
```

Comment:
remove path to dev environment

Reply:
I have pushed a change for this, thanks :)